orga/app/routes/join-when-authenticated.js

```diff
@@ -0,0 +1,20 @@
+import Route from '@ember/routing/route';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import { inject } from '@ember/service';
+
+export default Route.extend(AuthenticatedRouteMixin, {
+
+  session: inject(),
+
+  beforeModel(transition) {
+    this._super(...arguments);
+
+    const alternativeRoute = {
+      name: 'join',
+      queryParams: transition.to.queryParams
+    };
+    this.set('session.alternativeRoute', alternativeRoute);
```

Comment:
idem, avec 3.15, a priori, on peut faire:
```suggestion
    this.session.alternativeRoute = alternativeRoute;
```

Reply:
Done